frontend/next.config.js

```diff
@@ -15,6 +15,7 @@ const runtimeConfigs = {
     maxOnboardingAvailable: 3,
     featureFlags: {
       // Also add keys here to RuntimeConfig in src/config.ts
+      tips: false,
```

Comment:
@say-yawn If you could flip this to `true` and re-deploy to stage after you've done your deploy, then QA can still test it.